src/main/build/debug.h

```diff
@@ -112,6 +112,8 @@ typedef enum {
     DEBUG_CURRENT_ANGLE,
     DEBUG_DSHOT_TELEMETRY_COUNTS,
     DEBUG_RPM_LIMIT,
+    DEBUG_MAG_CALIB,
+    DEBUG_MAG_BIAS,
```

Comment:
I just noticed that all 8 debug values can be viewed in the configurator now, so maybe these modes can be merged into one.

Reply:
Thx. Current configurator build for windows has issues with Trojan:Win32/Wacatac.B!ml (its almost resolved already i think). Therefor I used an older configurator version which is not showing all 8 debugs.

Reply:
All fixed in the configurator, i will leave it as 2 debugs so that it is consistent with the testing process and the images from the PR description for now.